front/lib/api/csv.ts

```diff
@@ -1,5 +1,98 @@
 import { parse } from "csv-parse";
 
+const POSSIBLE_VALUES_MAX_LEN = 32;
+const POSSIBLE_VALUES_MAX_COUNT = 16;
+
+interface ColumnTypeInfo {
+  type: "string" | "number" | "boolean" | "enum";
```

Comment:
do we really need an "enum" type ? IMO, having the possible values is enough + you can have a finite/small set of possible values for columns that are of type "number" as well.

Reply:
I can change that

Reply:
Should be fixed!